readme.md

```diff
@@ -8,44 +8,39 @@
 
 ## About
 
-**Avalonia** is a WPF/UWP-inspired cross-platform XAML-based UI framework providing a flexible styling system and supporting a wide range of Operating Systems such as Windows (.NET Framework, .NET Core), Linux (via Xorg), macOS and with experimental support for Android and iOS.
+**Avalonia** is a cross-platform XAML-based UI framework providing a flexible styling system and supporting a wide range of Operating Systems such as Windows (.NET Framework, .NET Core), Linux (via Xorg), macOS.
```

Comment:
We probably should link to mobile section of the roadmap instead of removing mobile platfroms from readme entirely.

Reply:
Yeah I removed them because AFAIK they're just plain broken at the moment. I was thinking we could re-add them when we have at least a working prototype again...